README.md

```diff
@@ -2,6 +2,16 @@
 ## API Endpoints
 A description of all the API endpoints, their URL and request parameters.
 ### Users
+#### View User
+```
+url : /users/viewuser
```

Comment:
Change the url to something like /users/profile/
Not the forward slash at the end.

Reply:
Done